source/includes/_roles.md

```diff
@@ -0,0 +1,158 @@
+# Roles
+
+## Get All Roles
+
+```shell
+curl --request GET \
+  --url 'https://public-api.lumiformapp.com/api/v1/roles' \
```

Comment:
is Role a new endpoint we add recently?
if yes, why we do not add it to v2? 
it was not present in v1, as we develop v2 already, v1 needs to be frozen in the current state and all the additions go to v2.

Reply:
@seyfer No, it is not a new endpoint. For some reason there is no documentation for Role, but it is important for user v2 endpoint. So I included role documentation.